### DISCUSSION TITLE
iNavFlight 8.0.0. is not compatible with iNav 7.1.1 Configurator?

### DISCUSSION THREAD

**Discussion by mateo356** [iNavFlight/inav#10229] [Support]
Hi *,
After rebuilding the current state of the source code on GitHub (the last version in the CMake file is 8.0.0), it seems that the flight controller is unable to connect to the INav Configurator 7.1.1 and works only in CLI mode. Is there a new version of the iNav Configurator available?

Best regards

**Comment by MrD-RC** [comment]
That is how it should be. INAV firmware must be used with a Configurator of the same major version. 

If you are not a developer. You should be using INAV 7.1.2 with Configurator 7.1.1.

**Comment by breadoven** [comment]
There are nightly builds available at https://github.com/iNavFlight/inav?tab=readme-ov-file#developers.
